server/core/src/actors/v1_write.rs

```diff
@@ -1568,7 +1568,14 @@ impl QueryServerWriteV1 {
         let res = idms_prox_write
             .qs_write
             .purge_tombstones()
-            .and_then(|_| idms_prox_write.commit());
+            .and_then(|changed| {
+                // we don't need to commit the txn if there were no changes
```

Comment:
Actually we do, because we need this on tombstones *at least* to keep the RUV "touched". I'm working on this a bit more soon.

Reply:
Oh, I thought it was only commiting the TXN not doing... lots of other things. I'll back this out.